lib/plugins/awsResourcesDeploy/awsResourcesDeploy.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const AWS = require('aws-sdk');
+const _ = require('lodash');
+const path = require('path');
+const async = require('async');
+
+class AwsResourcesDeploy {
+  constructor(serverless) {
+    this.serverless = serverless;
+    this.options = {};
+    this.commands = {
+      resources: {
+        usage: 'deploys aws resources',
+        lifecycleEvents: [
+          'resources',
+        ],
+      },
+    };
+
+
+    this.hooks = {
+      'resources:resources': (options) => {
+        this.options = options;
+        return BbPromise.bind(this)
+          .then(this.validate)
+          .then(this.createOrUpdate)
+          .then(this.monitor)
+          .then(this.addOutputVariables)
+          .then(this.finish)
+          .catch((e) => {
+            // this.serverless.cli.spinner().stop();
+            throw new this.serverless.classes.Error(e);
+          });
+      },
+    };
+  }
+
+  validate() {
+    if (this.serverless.config.interactive && !this.options.noGreeting) {
+      this.serverless.cli.asciiGreeting();
+    }
+
+    if (!this.options.cfTemplate) {
+      throw new this.serverless.classes.Error('Please provide a cf template.');
+    }
+
+    if (!this.options.stage) {
+      throw new this.serverless.classes.Error('Please provide a stage name.');
+    }
+
+    if (!this.options.region) {
+      throw new this.serverless.classes.Error('Please provide a region name.');
+    }
+
+    // validate stage/region
+    this.serverless.service.getStage(this.options.stage);
+    this.serverless.service.getRegionInStage(this.options.stage, this.options.region);
+
+    // TODO validate region is valid aws
+
+    const config = {
+      region: this.options.region,
+    };
+    this.CloudFormation = new AWS.CloudFormation(config);
+    BbPromise.promisifyAll(this.CloudFormation, { suffix: 'Promised' });
+
+
+    console.log('');
+    this.serverless.cli.log('Deploying Resources to AWS...');
+    // this.serverless.cli.spinner().start(); - weird bug!
```

Comment:
Have you run `npm update`? I've removed the spinner and added it afterwards (was removed and re-added to the `package.json` during those commits).